injects/sampler.js

```diff
@@ -1,14 +1,20 @@
 'use strict'
 
 const fs = require('fs')
+const makeDir = require('mkdirp')
 const systemInfo = require('../collect/system-info.js')
 const ProcessStat = require('../collect/process-stat.js')
 const getLoggingPaths = require('../collect/get-logging-paths.js')
 const ProcessStatEncoder = require('../format/process-stat-encoder.js')
 
 // create encoding files and directory
-const paths = getLoggingPaths({ identifier: process.pid })
-fs.mkdirSync(paths['/'])
+let paths = null
+if (process.env.NODE_CLINIC_DOCTOR_DATA_PATH) {
+  paths = getLoggingPaths({ path: process.env.NODE_CLINIC_DOCTOR_DATA_PATH, identifier: process.pid })
+} else {
+  paths = getLoggingPaths({ identifier: process.pid })
+}
```

Comment:
this if should not be needed, we should be able to pass `path: process.env.NODE_CLINIC_DOCTOR_DATA_PATH` without an if.